### PR TITLE
Upgrade to CakePHP3.5

### LINKED ISSUE #127: We need Cake3.5
With current 3.4 and PHP7.2:
http://10.33.10.10/
=> Warning (2): count(): Parameter must be an array or an object that implements Countable [CORE/src/Database/QueryCompiler.php, line 128]

### PR DESCRIPTION
**Pull request by dereuromark**
This seems to resolve https://github.com/alt3/cakebox-console/issues/127
I tested the actions, looks good.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	"require": {
 		"php": ">=5.6",
 		"cakephp/bake": "~1.1",
-		"cakephp/cakephp": "3.4.*",
+		"cakephp/cakephp": "^3.5",
```
Comment (dereuromark) [inline composer.json:10]: Since we use a lock file, this should be fine here.

**Comment by bravo-kernel** [comment]
I cannot merge using phone if there are failed Travis results. It looks like phpcs is off somewhere. Otherwise this would have been merged straight in.

**Comment by dereuromark** [comment]
It is a behavior change in bootstrap for tests ( https://travis-ci.org/alt3/cakebox-console/jobs/345421127#L609 ) in 3.5.
So no biggy, should be enough to remove the DS from tests bootstrap.

**Comment by dereuromark** [comment]
Now follows https://github.com/cakephp/app/blob/master/config/paths.php#L18-L20

**Comment by bravo-kernel** [comment]
Could be my phone layout but I see some “tabs instead of spaces” phpcs fails

**Comment by dereuromark** [comment]
Yeah I see them too, in 3 year old code. Looks like a false positive of phpcs?

**Comment by bravo-kernel** [comment]
Weird stuff. Final check: you think this is good to merge?

**Comment by bravo-kernel** [comment]
Sorry, please ignore, I simply cannot merge using my phone as long as there is a travis fail, Github limitation.

Two choices: fix phpcs or wait until I am back at keyboard

**Comment by dereuromark** [comment]
I think we can wait :)

**Comment by bravo-kernel** [comment]
Thanks

**Pull request merged**